SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -1301,6 +1301,15 @@ SequenceableCollection : Collection {
 	// we break up the array so that missing elements are set to nil in the Quant
 	asQuant { ^Quant(*this) }
 
+	// Rational support
+	asRational { arg maxDenominator = 100;
```

Comment:
it's very obscure that an array of two numbers behaves differently than an array of three or more!